cmflib/commands/init/amazonS3.py

```diff
@@ -129,14 +129,14 @@ def add_parser(subparsers, parent_parser):
     required_arguments.add_argument(
         "--git-remote-url",
         required=True,
-        help="Specify git repo url.",
+        help="Specify git repo url. eg: https://github.com/XXX/example.git",
         metavar="<git_remote_url>",
         default=argparse.SUPPRESS,
     )
 
     parser.add_argument(
-        "--cmf-server-ip",
-        help="Specify cmf-server IP.",
+        "--cmf-server-url",
+        help="Specify cmf-server URL.",
```

Comment:
this cmf-server-ip which is now cmf-server-url is used above in code to read the value passed